examples/child_window.rs

```diff
@@ -0,0 +1,84 @@
+use std::collections::HashMap;
+
+use simple_logger::SimpleLogger;
+#[cfg(feature = "x11")]
+use winit::platform::unix::{WindowBuilderExtUnix, WindowExtUnix};
+use winit::{
+    dpi::{LogicalPosition, LogicalSize, Position},
+    event::{ElementState, Event, KeyboardInput, WindowEvent},
+    event_loop::{ControlFlow, EventLoop, EventLoopWindowTarget},
+    window::{Window, WindowBuilder},
+};
+
+#[cfg(feature = "x11")]
+fn spawn_child_window(
+    parent: u64,
+    event_loop: &EventLoopWindowTarget<()>,
+    windows: &mut HashMap<u64, Window>,
+) {
+    let child_window = WindowBuilder::new()
+        .with_x11_parent(parent)
```

Comment:
Need some help.

 In `examples/child_window.rs`, platform-specific functions configured to build only when `x11` feature but fails other feature like windows like this: https://github.com/rust-windowing/winit/runs/5859985935?check_suite_focus=true#step:9:129 but I don't know why...